src/Raven.Server/Documents/TransactionOperationsMerger.cs

```diff
@@ -86,7 +86,14 @@ public abstract class MergedTransactionCommand
             }
             finally
             {
-                _concurrentOperations.Signal(); // done with this
+                try
+                {
+                    _concurrentOperations.Signal(); // done with this
+                }
+                catch (InvalidOperationException)
+                {
+                    // Expected: "Invalid attempt made to decrement the event's count below zero."
```

Comment:
Note that this can happen if the server was shut down midway through the operation